modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -5,255 +5,258 @@
+[source, console]
+----
+mv owncloud owncloud-old-owncloud-version
+mkdir owncloud
+----
 
-NOTE: Don’t keep the `apps` directory.
 
-With those files and folders deleted, extract the archive of the latest
-ownCloud server, over the top of your existing installation.
+Extract the unpacked ownCloud server directory and its contents to the location of your original ownCloud installation.
 
 [source,console]
 ----
-# Extract the .tar.bz2 archive
-tar -jxf owncloud-10.0.3.tar.bz2 -C /var/www/
-
-# Extract the zip archive
-unzip -q owncloud-10.0.3.zip -d /var/www/
+tar -C /var/www/ -xvf owncloud-10.1.1.tar.bz2
 ----
 
-[[the-power-user-upgrade]]
-=== The Power User Upgrade
-
-Rename your current ownCloud directory, for example, from `owncloud` to `owncloud-old`. 
-Extract the unpacked ownCloud server directory and its contents to the location of your original ownCloud installation.
+With the new source files now in place of the old ones, next copy the `config.php` file from your old ownCloud directory to your new ownCloud directory. :
 
-IMPORTANT: These steps must be run as your web server user. 
-Otherwise,  you will likely encounter errors during the upgrade process, and ownCloud will likely not function correctly. 
-For more information, please refer to xref:installation/manual_installation.adoc#set-strong-directory-permissions[the details on setting strong directory permissions].
+[source,console]
+----
+sudo cp /var/www/owncloud-old/config/config.php /var/www/owncloud/config/config.php
+----
+* If you keep your `data/` directory _inside_ your `owncloud/` directory, move it from your old version of ownCloud to your new version:
 
+* If you keep it _outside_ of your `owncloud/` directory, then you don’t have to do anything with it, because its location is configured in your original `config.php`, and none of the upgrade steps touch it.
 [source,console]
 ----
-# Assumes that the new release was unpacked into /tmp/
-mv /tmp/owncloud /var/www/
+mv /var/www/owncloud-old-version-number/data /var/www/owncloud/data
 ----
+=== Market and Marketplace App Upgrades
+
+Before getting too far into the upgrade process, please be aware of how the Market app and its configuration options affect the upgrade process.
+
+* The Market app and other apps from the market place will not be updated when you upgrade owncloud and you configured `'upgrade.automatic-app-update' => true,` in your config.php.
 
-With the new source files now in place of the old ones, next copy the
-`config.php` file from your old ownCloud directory to your new ownCloud
-directory. :
+In addition, if there are installed apps (whether compatible or incompatible with the next version, or missing source code) and the Market app is enabled but there is no available internet connection, these apps will need to be manually updated once the upgrade is finished.
```

Comment:
This sentence works, but it is a bit long. If you feel inclined, it'd be worth trying to shorten it.